typings/type-utils.d.ts

```diff
@@ -0,0 +1,5 @@
+export type RequireAtLeastOne<T, Keys extends keyof T = keyof T> =
```

Comment:
Not sure I understand what this code is doing...

Also, the `utils` concept is not one that we want use. All software "should" have utility, so making a catch-all file, just makes purpose of this file hard to understand and less extensible.

Reply:
OK, I've renamed it. The purpose of this is to check that at least one method was describe. It can be applied not only for http methods, that is why I it was in file caled utils.